stl/inc/xmemory

```diff
@@ -606,7 +610,7 @@ struct _Normal_allocator_traits { // defines traits for allocators
 #endif // _HAS_IF_CONSTEXPR
 
 #if _HAS_IF_CONSTEXPR
-    _NODISCARD static size_type max_size(const _Alloc& _Al) noexcept {
+    _NODISCARD static _CONSTEXPR20_DYNALLOC size_type max_size(const _Alloc& _Al) noexcept {

```

Comment:
This requires allocator::max_size() to be `constexpr` or removed. I don't know why it wasn't removed by #380.

Reply:
I realised that it actually does not require it, because default allocator has it's own traits, but I still want to know why the deprecated `std::allocator` members weren't removed.

Reply:
Also `_Default_allocator_traits`s members need to be made constexpr.

Reply:
I am not sure why allocator::max_size() is there yet.
However, if it is problematic, could it be made `constexpr` with `_CONSTEXPR20_DYNALLOC` macro?